### PR TITLE
Add possibility of translate strings.

### LINKED ISSUE #21: Add translations
### Description

I tried to translate the "Add another" button within the Django administration. Unfortunately, it is not wrapped in a `{% trans %}` tag.

### PR DESCRIPTION
**Pull request by mondeja**
After this pull, users can extract and translate the two strings of the library. This closes #21.

### CODE REVIEW

**Comment by gradam** [comment]
Thanks for PR @mondeja 
Could you please fix formatting so that it follows black standards we have in this repo?

You could just run `black .` in the root dir of the project

File: django_better_admin_arrayfield/forms/fields.py [hunk 1]
```diff
@@ -27,7 +30,10 @@ def clean(self, value):
             except forms.ValidationError as error:
                 errors.append(
                     prefix_validation_error(
-                        error, self.error_messages["item_invalid"], code="item_invalid", params={"nth": index}
+                        error,
```
Comment (gradam) [inline django_better_admin_arrayfield/forms/fields.py:33]: Wrongly formatted

File: django_better_admin_arrayfield/forms/fields.py [hunk 2]
```diff
@@ -2,12 +2,15 @@
...
 from django.contrib.postgres.utils import prefix_validation_error
+from django.utils.translation import gettext_lazy as _
 from django_better_admin_arrayfield.forms.widgets import DynamicArrayWidget
 
 
 class DynamicArrayField(forms.Field):
 
-    default_error_messages = {"item_invalid": "Item %(nth)s in the array did not validate: "}
+    default_error_messages = {
+        "item_invalid":  _("Item %(nth)s in the array did not validate: "),
```
Comment (gradam) [inline django_better_admin_arrayfield/forms/fields.py:12]: wrongly formatted

Comment (mondeja) [reply django_better_admin_arrayfield/forms/fields.py]: After reformat with black shows me that this line is properly reformatted. What version of  `black` you are using?

**Comment by mondeja** [comment]
> Thanks for PR @mondeja
> Could you please fix formatting so that it follows black standards we have in this repo?
> 
> You could just run `black .` in the root dir of the project

I've reformatted the code with black and I've added `black==19.10b0` in `requiements_dev.txt` file.

**Comment by gradam** [comment]
@mondeja It's fine to pin it in the requirements file. thanks! However, I see that now there might be a mismatch of versions in said file and pre-commit. Could you set it in recommit as well?
Then I will merge it
Thanks!

**Pull request merged**